### PR TITLE
Added notifications on add to favorite and remove from favorite click.

### PR DESCRIPTION
**Pull request by martynaspuskunigis**

### CODE REVIEW

File: src/app/components/fav-recipe-item-view.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
 
-import { Recipe } from "./../contracts/Recipe";
 import { RecipesActionsCreators } from "./../recipes-actions-creators";
+import { Recipe } from "./../contracts/Recipe";
```
Comment (DeividasBakanas) [inline src/app/components/fav-recipe-item-view.tsx]: `./../contracts/Recipe` -> `../contracts/Recipe`

File: src/app/components/fav-recipe-item-view.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
 
-import { Recipe } from "./../contracts/Recipe";
 import { RecipesActionsCreators } from "./../recipes-actions-creators";
```
Comment (DeividasBakanas) [inline src/app/components/fav-recipe-item-view.tsx]: `./../recipes-actions-creators` -> `../recipes-actions-creators`

File: src/app/components/home-view.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 
-import { SearchView } from "./search-view";
 import { RecipesContainer } from "../recipes-container";
```
Comment (DeividasBakanas) [inline src/app/components/home-view.tsx]: Imports order.

File: src/app/components/recipes-item-view.tsx [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import * as React from "react";
+import * as Toastify from "react-toastify";
 import { Link } from "react-router-dom";
 
 import { RecipesActionsCreators } from "./../recipes-actions-creators";
```
Comment (DeividasBakanas) [inline src/app/components/recipes-item-view.tsx]: Simplify path.

File: src/app/components/recipes-item-view.tsx [hunk 5]
```diff
@@ -1,7 +1,9 @@
...
+
 import { Recipe } from "./../contracts/Recipe";
```
Comment (DeividasBakanas) [inline src/app/components/recipes-item-view.tsx]: Simplify path.

File: src/app/components/recipes-item-view.tsx [hunk 6]
```diff
@@ -12,8 +14,14 @@ interface Props {
...
+            Toastify.toast.success("Recipe removed from favorites", {
+                position: Toastify.toast.POSITION.BOTTOM_RIGHT
+            });
             RecipesActionsCreators.removeRecipeFromFavourites(recipeId);
         } else {
+            Toastify.toast.success("Recipe added to favorites", {
+                position: Toastify.toast.POSITION.BOTTOM_RIGHT
+            });
             RecipesActionsCreators.addRecipeToFavourites(recipeId);
         }
```
Comment (DeividasBakanas) [inline src/app/components/recipes-item-view.tsx]: Simplify condition. Single declaration `Toastify.toast.success` call should be enough.

File: src/app/styles/toastify.scss [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+    z-index: 9999;
+    position: fixed;
+    padding: 4px;
+    width: 320px;
+    box-sizing: border-box;
+    color: #fff; }
+    .Toastify__toast-container--top-left {
+      top: 1em;
+      left: 1em; }
+    .Toastify__toast-container--top-center {
```
Comment (DeividasBakanas) [inline src/app/styles/toastify.scss]: Is this a correnct formatting?

Comment (martynaspuskunigis) [reply src/app/styles/toastify.scss]: All these styles are default for the package I am using.

**Pull request merged**